src/lib/buildStatusMetricProps.ts

```diff
@@ -63,14 +68,14 @@ export function buildStatusMetricProps(
 
         if (!_.isFinite(value)) {
           displayValue = 'Invalid Number';
-        } else if (config.unit) {
-          displayValue = formattedValueToString(toFixedUnit(config.unit)(value, config.decimals));
+        } else if (config.unit) {          
```

Comment:
We should probably run prettier on this code to remove whitespace such as these

Reply:
good point, added a pre-commit hook for staged files. will run it one off in a separate PR, don't want too many unrelated formatting changes.